site/assets/scss/docs.scss

```diff
@@ -25,6 +25,10 @@
 
 // Load Bootstrap variables and mixins
 @import "../../../scss/functions";
+
+// fusv-disable
+$enable-responsive-pagination: true; // stylelint-disable-line scss/dollar-variable-default
```

Comment:
Not sure to understand here. It is enabled for all our paginations examples in the docs?